primefaces/src/main/resources/META-INF/resources/primefaces/core/core.css

```diff
@@ -1,8 +1,6 @@
 .ui-helper-hidden-accessible input,
 .ui-helper-hidden-accessible select {
-    -webkit-transform: scale(0); 
-    -moz-transform: scale(0); 
-    transform: scale(0);
+    transform: unset;
```

Comment:
also here, IMHO you can delete the rule completely instead of unsetting it

Reply:
I agree here.